cloud_testing/apis/testing/build.gradle

```diff
@@ -12,4 +12,5 @@ repositories {
 dependencies {
     // https://mvnrepository.com/artifact/com.google.api-client/google-api-client
     compile 'com.google.api-client:google-api-client:1.23.0'
+    compile 'com.google.cloud:google-cloud-nio:0.30.0-alpha'
```

Comment:
is this required to compile the API project?

Reply:
I think that may belong in the Kotlin build.gradle file instead:

[cloud_testing/kotlin_poc/build.gradle.kts#L64](https://github.com/TestArmada/flank/blob/88a4f8e5c61b25a8a795114ad295e59ef1803158/cloud_testing/kotlin_poc/build.gradle.kts#L64)